src/content/changes/highlights/2022-01-14.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2022-01-14T10:00:00
+title: Highlights for the week ending January 14, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+- [falco-app](https://github.com/giantswarm/falco-app) [v0.2.0](https://github.com/giantswarm/falco-app/blob/master/CHANGELOG.md#020---2021-12-17) Update to upstream charts: Falco 1.16.2/0.30.0, exporter 0.6.3/0.6.0, sidekick 0.4.4/2.24.0.
+- Dex monitoring is now available through our Public Grafana ( [Docs here](https://docs.giantswarm.io/advanced/configure-dex-in-your-cluster/) )
+
+## Documentation
+
+- Better clarification about Security offering compared to other cloud native packs
+- Use apiextensions-application v0.3.0 in CRD docs 
+
+## User interfaces
+
+Web UI
+ - KVM clusters can now be deleted without any issues from Happa
+ - Minor bugs that were patched:
+   1. Happa now correctly displays the number of upgradable apps
+   2. Consistency in displaying version labels
+   3. Organisations can now be deleted after moving to SSO
+
+Management API
+- [rbac-operator](https://github.com/giantswarm/rbac-operator) [v0.19.0](https://github.com/giantswarm/rbac-operator/blob/master/CHANGELOG.md#0190---2021-12-17) added user-friendly descriptions to created `ClusterRole` resources, via annotations using the `giantswarm.io/notes` key and changed the CI build process to use architect-orb.
+- [kyverno-policies](https://github.com/giantswarm/kyverno-policies) [v0.13.0](https://github.com/giantswarm/kyverno-policies/blob/master/CHANGELOG.md#0130---2022-01-05) added `policies-openstack` for OpenStack-specific policies and added policy for OpenStack which defaults `failureDomain` based on `MachineDeployment` request’s `machine-deployment.giantswarm.io/failure-domain` label.
```

Comment:
The Management API is not a user interface, so let's make it a separate section. That also simplifies the structure.

Also I suggest to remove some things from the user interfaces part.

Instead, adding the notice for kubectl-gs here again

Let's rather keep it short than force users to read through less interesting stuff.

```suggestion
## User interfaces

If you haven't done so yet, please upgrade your kubectl-gs version to 1.58.2, to prevent an issue with the `login` command when creating workload cluster certificates.

In the web UI on KVM installations, we fixed a problem that prevented clusters from being deleted. Also the number of apps with a pending upgrade is now calculated correctly.

## Management API

- [rbac-operator](https://github.com/giantswarm/rbac-operator) [v0.19.0](https://github.com/giantswarm/rbac-operator/blob/master/CHANGELOG.md#0190---2021-12-17) added user-friendly descriptions to created `ClusterRole` resources, via annotations using the `giantswarm.io/notes` key and changed the CI build process to use architect-orb.
- [kyverno-policies](https://github.com/giantswarm/kyverno-policies) [v0.13.0](https://github.com/giantswarm/kyverno-policies/blob/master/CHANGELOG.md#0130---2022-01-05) added `policies-openstack` for OpenStack-specific policies and added policy for OpenStack which defaults `failureDomain` based on `MachineDeployment` request’s `machine-deployment.giantswarm.io/failure-domain` label.
```